core/indexer/elasticsearchDatabaseClient.go

```diff
@@ -158,8 +159,19 @@ func (dc *databaseClient) DoMultiGet(obj object, index string) (object, error) {
 		return nil, fmt.Errorf("do multi get %s", res.String())
 	}
 
+	var responseBody []byte
+	responseBody, err = ioutil.ReadAll(res.Body)
+	if err != nil {
+		log.Warn("indexer: response body",
```

Comment:
You should delete this Warn, as responeBody should be nil if err != nil. Did you want to log res.Body? Also you can add it in the next warn as additional info:
log.Warn("indexer:cannot read from response body", "body", string(responseBody), "error", err)

Reply:
done.